subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -231,6 +229,37 @@ CeLoginRc checkServiceAuthorityAcfIntegrityV1(
     const uint64_t serialNumberLengthParm, ServiceAuthority& authorityParm,
     uint64_t& expirationTimeParm);
 
+/** @brief Extract metadata from an ACF
+ *
+ *  This function validates a few ACF fields, but only those minimally required to
+ *  read certain metadata fields. Full validation of the ACF will need to use the
+ *  upload interface.
+ *
+ *  @param accessControlFileParm a pointer to the ASN1 encoded binary ACF
+ *  @param accessControlFileLengthParm the length of the provided ACF
+ *  @param timeSinceUnixEpochInSecondsParm the current system time encoded as a unix timestamp
+ *  @param publicKeyParm a pointer to the public key used to vaidate the signature over the ACF
+ *  @param publicKeyLengthParm the length of the provided public key
+ *  @param serialNumberParm a pointer to the serial number of the current system
+ *  @param serialNumberLengthParm the length of the provided serial number
+ *  @param acfTypeParm the AcfType parsed from the provided file (e.g. service or admin reset)
+ *  @param expirationTimeParm the time at which this ACF expires
```

Comment:
A couple of missing params here... Could fix in the next pull request for the CLI